Scenes/Outline.shader

```diff
@@ -0,0 +1,56 @@
+shader_type spatial;
+render_mode unshaded;
+
+uniform float scale = 2.0;
+uniform float threshold = 2.0;
+uniform vec3 color = vec3(0, 0, 0);
+uniform bool enabled = true;
+
+varying mat4 CAMERA;
+
+void vertex() 
+{
+	POSITION = vec4(VERTEX, 1.0);
+	CAMERA = CAMERA_MATRIX;
+}
+
+float depth_calc(vec2 uv, sampler2D depth_tex, mat4 inv_projection_mat) 
```

Comment:
why don't just use `DEPTH_TEXTURE` and `INVERSE_PROJECTION_MATRIX` directly?

Reply:
For some reason cannot access from the function. Quite strange.

Reply:
Oh orkei